modules/programs/gh.nix

```diff
@@ -91,11 +97,21 @@ in {
       '';
     };
 
-    enableGitCredentialHelper =
-      mkEnableOption "the gh git credential helper for github.com" // {
+    gitCredentialHelper = {
+      enable = mkEnableOption "the gh git credential helper" // {
         default = true;
       };
 
+      hosts = mkOption {
+        type = types.listOf types.str;
+        default = [ "https://github.com" ];
```

Comment:
Maybe move this default value to `config`, so that people can append to the list while keeping the default value (and still be able to `mkForce` if they want to remove github)

Reply:
On second thought I'm not sure about this, it creates ambiguity since anyone who sets the `hosts` option now intuitively can't be sure whether they also need to include `"https://github.com"` in it.

I'd argue that those people who do actually end up adding additional hosts (which is probably a small subset of users of the module) will probably think about ensuring the default is still there.

Additionally, I'd find it confusing if the option is called `hosts` but doesn't actually have to include all the hosts that you enable the option for.

Reply:
Fair enough